<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>953</width>
                <height>669</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <item>
                            <widget class="QListWidget" name="listWidget">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="GraphWidget" name="graphWidget" native="true"/>
                        </item>
                    </layout>
                </item>
                <item row="1" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QLabel" name="currentTimeLabel">
                                <property name="text">
                                    <string>0:00</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QSlider" name="horizontalSlider">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="totalTimeLabel">
                                <property name="text">
                                    <string>0:00</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item row="2" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                            <widget class="QPushButton" name="showButton">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Show</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer_3">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeType">
                                    <enum>QSizePolicy::Expanding</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item alignment="Qt::AlignHCenter">
                            <widget class="QPushButton" name="playButton">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Play</string>
                                </property>
                                <attribute name="buttonGroup">
                                    <string notr="true">buttonGroup</string>
                                </attribute>
                            </widget>
                        </item>
                        <item alignment="Qt::AlignHCenter">
                            <widget class="QPushButton" name="stopButton">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Stop</string>
                                </property>
                                <attribute name="buttonGroup">
                                    <string notr="true">buttonGroup</string>
                                </attribute>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeType">
                                    <enum>QSizePolicy::MinimumExpanding</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="equalizerButton">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Equalizer</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer_2">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeType">
                                    <enum>QSizePolicy::Fixed</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="volumeButton">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Mute</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QSlider" name="volumeSlider">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>953</width>
                    <height>26</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionOpen_File"/>
            </widget>
            <addaction name="menuFile"/>
        </widget>
        <action name="actionOpen_File">
            <property name="text">
                <string>Open File</string>
            </property>
        </action>
    </widget>
    <customwidgets>
        <customwidget>
            <class>GraphWidget</class>
            <extends>QWidget</extends>
            <header>Graph</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
    <buttongroups>
        <buttongroup name="buttonGroup"/>
    </buttongroups>
</ui>
